scene/resources/default_theme/default_theme.cpp

```diff
@@ -569,9 +569,10 @@ void fill_default_theme(Ref<Theme> &theme, const Ref<Font> &default_font, const
 	Ref<StyleBoxTexture> style_pp = sb_expand(make_stylebox(popup_bg_png, 5, 5, 5, 5, 4, 4, 4, 4), 2, 2, 2, 2);
 
 	Ref<StyleBoxTexture> selected = make_stylebox(selection_png, 6, 6, 6, 6);
-	for (int i = 0; i < 4; i++) {
-		selected->set_expand_margin_size(Margin(i), 2 * scale);
-	}
+	selected->set_expand_margin_size(MARGIN_TOP, 2 * scale);
+	selected->set_expand_margin_size(MARGIN_BOTTOM, 2 * scale);
+	selected->set_expand_margin_size(MARGIN_LEFT, 6 * scale);
+	selected->set_expand_margin_size(MARGIN_RIGHT, 6 * scale);
 
```

Comment:
And this seems correct/needed too.

- Default theme in `3.5.rc3` (with that excessive `hover->draw(...)` call left by me):
![Godot_v3 5-rc3_win64_tVrJOdNg6P](https://user-images.githubusercontent.com/9283098/172005464-d7556241-4bcb-42e8-961d-1739d888c73a.png)
It clearly shows the fixed `hover->draw(...)` call results in a smaller rect for the default theme.

- Default theme in this PR with the padding added:
![godot windows opt tools 64_eW0ssAz2BY](https://user-images.githubusercontent.com/9283098/172005730-911122df-d062-473c-9a1a-a1b618a4a16f.png)